mobile/src/bundles/common-data/store/actions.ts

```diff
@@ -0,0 +1,34 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { ApiPath } from 'shared/build/index';
+
+import { type AsyncThunkConfig } from '~/bundles/common/types/types';
+import {
+    type BadgesResponseDto,
+    type HardSkillsResponseDto,
+} from '~/bundles/common-data/types/types';
+
+import { name as sliceName } from './slice';
+
+const getHardSkillsData = createAsyncThunk<
+    HardSkillsResponseDto,
+    undefined,
+    AsyncThunkConfig
+>(`${sliceName}${ApiPath.HARD_SKILLS}`, async (_, { extra }) => {
+    const { commonDataApi } = extra;
+
+    return commonDataApi.getHardSkillsData();
+    // TODO: write error to logger
```

Comment:
what do you mean in this Todo?

Reply:
thougnt that we shall create server logger with errors and so on, are we?

Reply:
I think no, we don't need to notify user about any error with common data, because it's not a user error that data is not loading, that's why I said to remove toast